File: MatchUpAPI/src/main/java/com/matchup/api/matchup_api/models/Sport.java
package com.matchup.api.matchup_api.models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

import java.util.UUID;

@Data
@Entity
@Table(name = "Sports")
public class Sport {
    @Id
    @Column(name = "spo_id")
    private UUID id;
    @Column(name = "spo_name")
    private String name;
    @Column(name = "spo_icon")
    private byte[] icon = null;
}

File: MobileApp/app/src/main/java/com/app/matchup/models/Sport.kt
package com.app.matchup.models

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.util.UUID

@Parcelize
data class Sport (
    val id: UUID = UUID.randomUUID(),
    val name: String,
    var icon: Int? = null
): Parcelable
